src/script/storage.ts

```diff
@@ -0,0 +1,28 @@
+import { Post } from './api.ts';
```

Comment:
We can use `type` here to optimize this. 
```suggestion
import type { Post } from './api.ts';
```

Reply:
add https://github.com/Sakshi-Simform/product-management/pull/3/commits/9c5516df96f609dd01b52a749d2f3637e6cff67b